lxd/instance.go

```diff
@@ -612,12 +629,20 @@ func pruneExpiredContainerSnapshotsTask(d *Daemon) (task.Func, task.Schedule) {
 }
 
 func pruneExpiredContainerSnapshots(ctx context.Context, d *Daemon, snapshots []instance.Instance) error {
```

Comment:
Lets rename to `pruneExpiredInstanceSnapshots`

Reply:
Fixed.